docs/plain-bundle-spec.md

```diff
@@ -210,3 +212,9 @@ spec:
   provisionerClassName: core.rukpak.io/plain
 EOF
 ```
+
+#### Method 2: Add the secret to the `imagePullSecrets` in the `default` service account in the privisioner deployed namespace
+
+```bash
+kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "myrsecret"}]}' -n rukpak-system
```

Comment:
```suggestion
kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "mysecret"}]}' -n rukpak-system
```

Does this patch merge the secret with other potential secrets referenced by the SA, or do they get dropped?

Reply:
It's a good question.  This replaces the secrets already in the service account.  I'll add a  note about it.  Thanks!